### PR TITLE
move implicit templates into its own path

### PR DESCRIPTION
**Pull request by mwellman17**

### CODE REVIEW

File: src/templates.rs [hunk 1]
```diff
@@ -72,12 +72,10 @@ impl Templates {
...
         template_name: &str,
     ) -> GraphQLResult<Option<String>> {
-        let name_prefix: String = "_getit_".to_owned();
-        let full_name = name_prefix + template_name;
         let query = GetImplicitTemplateQuery::build_query(get_implicit_template_query::Variables {
             organization_id: organization_id.map(|id| id.to_string()),
             environment_name: environment_name.map(|name| name.to_string()),
-            template_name: full_name,
+            template_name: template_name.to_string(),
             filters: get_implicit_template_query::ImplicitTemplateFilters {
```
Comment (rickporter-tuono) [inline src/templates.rs:79]: Why do we need a template name at all in the query?

Comment (mwellman17) [reply src/templates.rs:79]: the template_name is really the template format, one of "dotenv", "docker", or "shell"

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ jobs:
         with:
           name: Clippy Check
           token: ${{ secrets.GITHUB_TOKEN }}
-          args: --all-features -- -D warnings
+          args: --all-features -- -D warnings -A clippy::upper_case_acronyms
```
Comment (rickporter-tuono) [inline .github/workflows/ci.yml:49]: Although this seems to work for CI, I don't think it is the right way. I think we want a crate-level disabling, so it applies when you run `cargo clippy` outside of our CI workflow.

https://stackoverflow.com/questions/25877285/how-to-disable-unused-code-warnings-in-rust

Comment (nirvdrum) [reply .github/workflows/ci.yml:49]: I agree. I think we're better off just creating a [clippy.toml](https://github.com/rust-lang/rust-clippy#configuration) at the project root and provide any global overrides in there. Disabling just for CI won't help with pre-commit hooks, for instance.

Comment (mwellman17) [reply .github/workflows/ci.yml:49]: unfortunately clippy.toml only applies to lints with configurable options, not individual lints https://www.reddit.com/r/rust/comments/eyjq12/clippy_configuration_options/fghuckg?utm_source=share&utm_medium=web2x&context=3

Hopefully Rick's PR will work for us and I'll remove this argument.

**Pull request merged**